charts/rd-judicial-api/values.yaml

```diff
@@ -13,7 +13,11 @@ java:
     OPEN_ID_API_BASE_URI: https://idam-web-public.{{ .Values.global.environment }}.platform.hmcts.net/o
     LAUNCH_DARKLY_ENV: "{{ .Values.global.environment }}"
     ENVIRONMENT_NAME: "{{ .Values.global.environment }}"
-  image: 'hmctspublic.azurecr.io/rd/judicial-api:latest'
+    image: 'hmctspublic.azurecr.io/rd/judicial-api:latest'
+    LOCATION_REF_DATA_URL: http://rd-location-ref-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+    REFRESH_PAGE_SIZE: 20
```

Comment:
page size and perid can be kept at application.yml. Environmental level variables are preferred to be kept here.